### DISCUSSION TITLE
plot min/max values on legend

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1118] [Questions / Documentation]
Hello,

It seems like `plotEmbedding()` is a wrapper to generate a ggplot. If I want to `colorBy` my `GeneExpressionMatrix`, how would I manipulate this object to only plot the minimum and maximum values in the legend? I would like to do something like [this](https://stackoverflow.com/questions/62842840/setting-min-max-size-in-legend-in-ggplot) but am not sure how to interact with the values.

Thank you

**Comment by rcorces** [comment]
I believe you are looking for the `colorLimits` parameter. You would have to determine the min and max values from the matrix and then pass them to `colorLimits`

<html>
<body>


colorLimits | A numeric vector of two values indicating the lower and upper bounds of colors if numeric. Values beyond these limits are thresholded.
-- | --



</body>
</html>
